app/services/reports/region_service.rb

```diff
@@ -4,7 +4,7 @@ class RegionService
 
     # The default period we report on is the current month.
     def self.default_period
-      Period.month(Date.current.beginning_of_month)
+      Period.month(Date.current.in_time_zone(Period::ANALYTICS_TIME_ZONE))
```

Comment:
Do we need `beginning_of_month`?

Reply:
> Do we need `beginning_of_month`?

yes we definitely want beginning of month here.

Reply:
`Period.month` already does a `beginning_of_month`. ex. Period.month(Date.today) = 1st May. It's not _required_, no?

Reply:
Ah nice. Yup, if `Period.month` takes care of `beginning_of_month`, then we should be good 🆗